keyboards/quark/keymaps/ajp10304/rules.mk

```diff
@@ -0,0 +1 @@
+MOUSEKEY_ENABLE = yes
```

Comment:
Adding `DYNAMIC_MACRO_ENABLE = yes` in here means the board stops working completely.  It works in my other keymaps.  Any idea why the Quark would be different?

Reply:
Dynamic macros uses a lot of memory.  My guess is that the chip doesn't have enough for everything enabled.

Reply:
Hmm, I did wonder about that. Thanks for the reply.